auth/builtin.go

```diff
@@ -62,25 +62,14 @@ func (r *NoAuth) Refresh(refresh string) (token Token, err error) {
 type Builtin struct {
 }
 
-// Parse Token out of a string
-func parseToken(requestToken string) (token string, err error) {
-	splitToken := strings.Fields(requestToken)
-	if len(splitToken) != 2 || strings.ToLower(splitToken[0]) != "bearer" {
-		err = liberr.Wrap(&NotValid{Token: requestToken})
-		return
-	}
-	token = splitToken[1]
-	return
-}
-
```

Comment:
**hint**: No reason for parseToken() to be a package function. Consistent with the project, this should be a method on Builtin.
Better for parameter to be a request.